example/index.spec.ts

```diff
@@ -2,18 +2,18 @@ import assert from "node:assert/strict";
 import { EventSource } from "undici";
 import { spawn } from "node:child_process";
 import { createReadStream, readFileSync } from "node:fs";
-import { Client, Subscription } from "../../example/example.client";
-import { givePort } from "../helpers";
+import { Client, Subscription } from "./example.client";
+import { givePort } from "../tests/helpers";
 import { createHash } from "node:crypto";
 import { readFile } from "node:fs/promises";
 
-describe("Example", async () => {
+describe("System test for Example", async () => {
   let out = "";
   const listener = (chunk: Buffer) => {
     out += chunk.toString();
   };
   const matchOut = (regExp: RegExp) => regExp.test(out);
-  const example = spawn("tsx", ["example/index.ts"]);
+  const example = spawn("tsx", ["index.ts"]);
```

Comment:
some issue with it in CI (exclusively)

Reply:
ah , yes!

```
Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/home/runner/work/express-zod-api/express-zod-api/node_modules/express-zod-api/dist/index.js' imported from /home/runner/work/express-zod-api/express-zod-api/example/index.ts
```

we need to build it first because example is now using the distribution